projects/igniteui-angular/src/lib/core/styles/components/carousel/_carousel-theme.scss

```diff
@@ -286,12 +301,15 @@
     }
 
     %igx-nav-dot--active {
-        border: $indicator-border-style;
-        border-color: --var($theme, 'indicator-active-border-color');
+        @if $variant != 'indigo-design' {
+            border: $indicator-border-style;
+            border-color: --var($theme, 'indicator-active-border-color');
+        }
+
         opacity: 1;
 
         &::after {
-            opacity: 1;
+            background: --var($theme, 'indicator-active-dot-color');
             @include animation('scale-in-center' .15s $ease-out-quad forwards);
```

Comment:
I think this animation should also only be included when the theme is not 'indigo-design'.